2022-01-26 15:45:00,739 INFO    MainThread:1026140 [wandb_setup.py:_flush():71] setting env: {}
2022-01-26 15:45:00,739 WARNING MainThread:1026140 [wandb_setup.py:_flush():71] could not save program above cwd: /home/aadi/projects/pgrad-thesis/src/runall.py
2022-01-26 15:45:00,739 INFO    MainThread:1026140 [wandb_setup.py:_flush():71] setting login settings: {}
2022-01-26 15:45:00,739 INFO    MainThread:1026140 [wandb_init.py:_log_setup():371] Logging user logs to /storage/projects/pgrad-thesis/wandb/run-20220126_154500-r2goqrlw/logs/debug.log
2022-01-26 15:45:00,739 INFO    MainThread:1026140 [wandb_init.py:_log_setup():372] Logging internal logs to /storage/projects/pgrad-thesis/wandb/run-20220126_154500-r2goqrlw/logs/debug-internal.log
2022-01-26 15:45:00,739 INFO    MainThread:1026140 [wandb_init.py:init():404] calling init triggers
2022-01-26 15:45:00,739 INFO    MainThread:1026140 [wandb_init.py:init():409] wandb.init called with sweep_config: {}
config: {}
2022-01-26 15:45:00,739 INFO    MainThread:1026140 [wandb_init.py:init():460] starting backend
2022-01-26 15:45:00,739 INFO    MainThread:1026140 [backend.py:_multiprocessing_setup():99] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-01-26 15:45:00,747 INFO    MainThread:1026140 [backend.py:ensure_launched():216] starting backend process...
2022-01-26 15:45:00,751 INFO    MainThread:1026140 [backend.py:ensure_launched():221] started backend process with pid: 1026198
2022-01-26 15:45:00,752 INFO    MainThread:1026140 [wandb_init.py:init():469] backend started and connected
2022-01-26 15:45:00,754 INFO    MainThread:1026140 [wandb_init.py:init():533] updated telemetry
2022-01-26 15:45:00,764 INFO    MainThread:1026140 [wandb_init.py:init():563] communicating current version
2022-01-26 15:45:01,294 INFO    MainThread:1026140 [wandb_init.py:init():568] got version response 
2022-01-26 15:45:01,294 INFO    MainThread:1026140 [wandb_init.py:init():578] communicating run to backend with 30 second timeout
2022-01-26 15:45:01,710 INFO    MainThread:1026140 [wandb_init.py:init():606] starting run threads in backend
2022-01-26 15:45:04,385 INFO    MainThread:1026140 [wandb_run.py:_console_start():1810] atexit reg
2022-01-26 15:45:04,386 INFO    MainThread:1026140 [wandb_run.py:_redirect():1684] redirect: SettingsConsole.REDIRECT
2022-01-26 15:45:04,386 INFO    MainThread:1026140 [wandb_run.py:_redirect():1689] Redirecting console.
2022-01-26 15:45:04,387 INFO    MainThread:1026140 [wandb_run.py:_redirect():1745] Redirects installed.
2022-01-26 15:45:04,387 INFO    MainThread:1026140 [wandb_init.py:init():633] run started, returning control to user process
2022-01-26 15:45:08,908 INFO    MainThread:1026140 [wandb_run.py:_atexit_cleanup():1780] got exitcode: 1
2022-01-26 15:45:08,909 INFO    MainThread:1026140 [wandb_run.py:_restore():1752] restore
2022-01-26 15:45:11,391 INFO    MainThread:1026140 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 754
  total_bytes: 754
}

2022-01-26 15:45:11,770 INFO    MainThread:1026140 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 754
  total_bytes: 754
}

2022-01-26 15:45:12,607 INFO    MainThread:1026140 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 754
  total_bytes: 68942
}

2022-01-26 15:45:12,708 INFO    MainThread:1026140 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 754
  total_bytes: 68942
}

2022-01-26 15:45:12,809 INFO    MainThread:1026140 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 754
  total_bytes: 68942
}

2022-01-26 15:45:12,912 INFO    MainThread:1026140 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 754
  total_bytes: 68942
}

2022-01-26 15:45:13,014 INFO    MainThread:1026140 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 754
  total_bytes: 68942
}

2022-01-26 15:45:13,115 INFO    MainThread:1026140 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 68942
  total_bytes: 68942
}

2022-01-26 15:45:13,216 INFO    MainThread:1026140 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 68942
  total_bytes: 68942
}

2022-01-26 15:45:13,317 INFO    MainThread:1026140 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 68942
  total_bytes: 68942
}

2022-01-26 15:45:13,419 INFO    MainThread:1026140 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 68942
  total_bytes: 68942
}

2022-01-26 15:45:13,520 INFO    MainThread:1026140 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 68942
  total_bytes: 68942
}

2022-01-26 15:45:13,621 INFO    MainThread:1026140 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 68942
  total_bytes: 68942
}

2022-01-26 15:45:13,865 INFO    MainThread:1026140 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 68942
  total_bytes: 68942
}

2022-01-26 15:45:14,503 INFO    MainThread:1026140 [wandb_run.py:_wait_for_finish():1912] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 68942
  total_bytes: 68942
}
local_info {
}

2022-01-26 15:45:15,674 INFO    MainThread:1026140 [wandb_run.py:_append_files():2180] logging synced files
